### PR TITLE
[No-30][Training]Get code laravel dusk

### PR DESCRIPTION
**Pull request by loitq**
- Get code laravel dusk installed.
- Remove file example.

### CODE REVIEW

**Review by phantuan1803 — APPROVED** [review]
LGTM

**Comment by ghost** [comment]
LGTM

**Review by ghost — APPROVED** [review]

File: .env.testing [hunk 1]
```diff
@@ -0,0 +1,44 @@
+APP_NAME=Laravel
```
Comment (hienl) [inline .env.testing]: This app not name Lavarel

**Review by hienl — CHANGES_REQUESTED** [review]
Please Fix

File: .env.testing (hunk 1 above)
Comment (loitq) [reply .env.testing]: I fixed

**Review by hienl — APPROVED** [review]
LGTM

**Pull request merged**